Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Editor/CompositorWindow.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Editor
+{
+    [Description("Compositor")]
+    public class CompositorWindow : SpectatorViewWindowBase<CompositorWindow>
+    {
+        private const float maxFrameOffset = 0.2f;
+        private const float statisticsUpdateCooldownTime = 0.1f;
```

Comment:
>statisticsUpdateCooldownTime [](start = 28, length = 28)

Nit: Add a comment about the units (in this case, I think seconds)

Reply:
Or use TimeSpan :)

---
In reply to: [282565567](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4246#discussion_r282565567) [](ancestors = 282565567)

Reply:
Added "Seconds" to the names of both time-related variables to make the code easier to understand